Makefile

```diff
@@ -0,0 +1,90 @@
+.PHONY: all build clean test lint vet fmt image push
+
+# Go parameters
+GOCMD=go
+GOBUILD=$(GOCMD) build
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+GOGET=$(GOCMD) get
+GOMOD=$(GOCMD) mod
+GOLINT=golangci-lint
+GOVET=$(GOCMD) vet
+GOFMT=$(GOCMD) fmt
+
+GOOS=$(shell go env GOOS)
+GOARCH=$(shell go env GOARCH)
+
+# Project parameters
+BINARY_NAME=kepler
+BINARY_DIR=bin
+MAIN_GO_PATH=./cmd/kepler
+VERSION=$(shell git describe --tags --always --dirty || echo "dev")
```

Comment:
@vprashar2929 we want the version to have a constant prefix, say k2-$(VERSION) ?